src/components/TaxReport/TaxReport.js

```diff
@@ -176,22 +196,26 @@ class TaxReport extends PureComponent {
       t,
     } = this.props
     const {
-      winLossTotalAmount,
-      startPositionsSnapshot,
-      startTickers,
-      endPositionsSnapshot,
-      endTickers,
-      movementsEntries,
-      depositsTotalAmount,
-      withdrawalsTotalAmount,
-      movementsTotalAmount,
+      startingPositionsSnapshot,
+      endingPositionsSnapshot,
+      finalState: {
+        startingPeriodBalances,
+        endingPeriodBalances,
+        movements,
+        movementsTotalAmount,
+        totalResult,
+      },
     } = data
     const { start, end } = this.state
     const hasNewTime = this.hasNewTime()
 
-    const isEmpty = !_isNumber(winLossTotalAmount) && !startPositionsSnapshot.length && !endPositionsSnapshot.length
-      && !movementsEntries.length && !_isNumber(depositsTotalAmount)
-      && !_isNumber(withdrawalsTotalAmount) && !_isNumber(movementsTotalAmount)
+    const isEmpty = !startingPositionsSnapshot.length
+      && !endingPositionsSnapshot.length
+      && this.isBalancesEmpty(startingPeriodBalances)
+      && this.isBalancesEmpty(endingPeriodBalances)
+      && !movements.length
```

Comment:
I dont get this, if a certain report suddenly dont have movements it would be consider empty?

Reply:
Only if every piece of data is empty, that's to show `No related data in this time range` message.